src/server/server.ts

```diff
@@ -47,6 +47,15 @@ function isOfficeHost(host: string) {
     return officeHosts.indexOf(host) >= 0;
 }
 
+let assetPaths = null;
+function getAssetPaths() {
+    if (assetPaths === null) {
+        let data = fs.readFileSync(path.resolve(__dirname, 'assets.json'));
+        assetPaths = JSON.parse(data.toString());
+    }
+
+    return assetPaths;
+}
 
```

Comment:
OK, fair enough.

---
In reply to: [126569799](https://github.com/OfficeDev/script-lab/pull/474#discussion_r126569799) [](ancestors = 126569799)